site/about/glossary/_validmind.qmd

```diff
@@ -0,0 +1,9 @@
+#### ValidMind AI risk platform (ValidMind)
```

Comment:
```suggestion
#### ValidMind AI risk platform
```

ValidMind appears three times on two lines, this line and the heading immediately preceding it. We can likely omit at least one of these and still get the point across.